internal/cli/organizations.go

```diff
@@ -497,9 +508,10 @@ func listMembersOrganizationCmd(cli *cli) *cobra.Command {
 		Aliases: []string{"ls"},
 		Args:    cobra.MaximumNArgs(1),
 		Short:   "List members of an organization",
-		Long:    "List members of an organization.",
-		Example: `auth0 orgs members list
-auth0 orgs members ls <id>`,
+		Long:    "List the members of an organization.",
+		Example: `  auth0 orgs members list
+  auth0 orgs members ls <id>
+  auth0 orgs members ls <id> --json`,
```

Comment:
As an org is likely to have many members, consider in a future PR adding an example with the `-n` flag, like in other similar commands.